apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx

```diff
@@ -224,61 +234,44 @@ export const RulingStepOne: React.FC = () => {
                       ? new Date(workingCase.requestedCustodyEndDate)
                       : null
                   }
-                  handleChange={(date) => {
-                    const formattedDate = formatISO(date, {
-                      representation: workingCase.custodyEndDate?.includes('T')
-                        ? 'complete'
-                        : 'date',
-                    })
-
-                    setWorkingCase({
-                      ...workingCase,
-                      custodyEndDate: formattedDate,
-                    })
-
-                    updateCase(
-                      workingCase.id,
-                      parseString('custodyEndDate', formattedDate),
+                  handleChange={(date) =>
+                    setAndSendDateToServer(
+                      'custodyEndDate',
+                      workingCase.custodyEndDate,
+                      date,
+                      workingCase,
+                      setWorkingCase,
+                      updateCase,
                     )
-                  }}
+                  }
                   required
                 />
               </GridColumn>
               <GridColumn span="5/12">
                 <TimeInputField
-                  onFocus={() => setCustodyEndTimeErrorMessage('')}
-                  onBlur={(evt) => {
-                    const time = padTimeWithZero(evt.target.value)
-
-                    if (workingCase.custodyEndDate) {
-                      const validateTimeEmpty = validate(time, 'empty')
-                      const validateTimeFormat = validate(time, 'time-format')
-                      const custodyEndDateMinutes = parseTime(
-                        workingCase.custodyEndDate,
-                        time,
-                      )
-
-                      setWorkingCase({
-                        ...workingCase,
-                        custodyEndDate: custodyEndDateMinutes,
-                      })
-
-                      if (
-                        validateTimeEmpty.isValid &&
-                        validateTimeFormat.isValid
-                      ) {
-                        updateCase(
-                          workingCase.id,
-                          parseString('custodyEndDate', custodyEndDateMinutes),
-                        )
-                      } else {
-                        setCustodyEndTimeErrorMessage(
-                          validateTimeEmpty.errorMessage ||
-                            validateTimeFormat.errorMessage,
-                        )
-                      }
-                    }
-                  }}
+                  onChange={(evt) =>
```

Comment:
Why do we no longer need onFocus? Both here and elsewhere.

Reply:
The only thing onFocus was used for was to remove the error message. Now the error message gets removed when the field becomes valid.